src/ensembl/src/shared/components/error-screen/GeneralErrorScreen.tsx

```diff
@@ -14,34 +14,84 @@
  * limitations under the License.
  */
 
-import React from 'react';
+import storageService from 'ensemblRoot/src/services/storage-service';
```

Comment:
- 🙅‍♂️ `ensemblRoot`
- we put npm imports (react, classNames) first

Reply:
Oh no! again!
Btw I have already reordered the imports :-)